src/vm/eventpipe.h

```diff
@@ -307,13 +324,18 @@ class EventPipe
         static EventPipeConfiguration *s_pConfig;
         static EventPipeSession *s_pSession;
         static EventPipeBufferManager *s_pBufferManager;
+        static LPCWSTR s_pOutputPath;
+        static unsigned long s_nextFileIndex;
```

Comment:
I guess it is OK right now to make this a static (since everything else is as well), but it would have been just as easy to make it a member of the session so there was just that much less to move over to the session when we support multi-session.